### PR TITLE
Updated CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by bvitaliyg**
+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] Component `README.md`
+ [ ] Tests
+ [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -45,8 +50,15 @@ Given that you have a compatible environment as stated above you can now set up
 #### Snapshot testing
 Create an AVD using the following command
 
+> Note: Currently, screenshot testing doesn't work properly on M1 chips.
```
Comment (marianeum) [inline CONTRIBUTING.md:53]: Maybe add something what to do in this case (eg generate screenshot tests on a different machine/check with another engineer if not available)

Comment (bvitaliyg) [reply CONTRIBUTING.md:53]: This solution should come as an output of the ticket I created :)

Comment (marianeum) [reply CONTRIBUTING.md:53]: yeah but in the meantime we need have the right instructions - particularly with m1s rolled out more widely soon

Comment (bvitaliyg) [reply CONTRIBUTING.md]: Done

**Review by marianeum — APPROVED** [review]

**Pull request merged**